velox/functions/iceberg/tests/DateTimeFunctionsTest.cpp

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/common/base/tests/GTestUtils.h"
+#include "velox/functions/iceberg/tests/IcebergFunctionBaseTest.h"
+#include "velox/type/Timestamp.h"
```

Comment:
"Timestamp.h" might not be needed, as it is pulled in through "IcebergFunctionBaseTest.h".

Reply:
No, I noticed this because I assumed "Timestamp.h" is basic and would already be included by "Type.h". There’s also a tool designed for this purpose: https://include-what-you-use.org.

Reply:
Yes, I tried it a long times ago, it will format all the velox files, maybe we should need a check for the new changed  files, it's also ok for me to rely on committer to check the header.